ambry-store/src/main/java/com.github.ambry.store/DiskSpaceRequirements.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.store;
+
+/**
+ * Describes the additional segments required by a store-related entity. This is provided to the
+ * {@link DiskSpaceAllocator} to initialize a disk space pool.
+ */
+class DiskSpaceRequirements {
+  private final long segmentSizeInBytes;
```

Comment:
Because the DiskSpaceAllocator aggregates all of the DiskSpaceRequirements into one set of requirements, I don't think we can tell whether one of the DiskSpaceRequirements is invalid (if they were constructed successfully (i.e. no negative values)). I can actually add debug logging to the getDiskSpaceRequirements method in blob store, instead.